project/Dependencies.scala

```diff
@@ -263,6 +263,13 @@ object Dependencies {
     )
   )
 
+  val Stomp = Seq(
+    libraryDependencies ++= Seq(
+      // https://mvnrepository.com/artifact/io.vertx/vertx-stomp
+      "io.vertx" % "vertx-stomp" % "3.5.1" // ApacheV2/EPL1
```

Comment:
Do we have a sense of the dependencies here ie is it bringing in many?

Reply:
What’s the thread usage of this dependency in terms of the number of threads required?

Reply:
I’ve implemented STOMP before - the protocol is easy. Should we even need this dependency?

Reply:
> Do we have a sense of the dependencies here ie is it bringing in many?

It brings stuff from vertx like vertx-core

> What’s the thread usage of this dependency in terms of the number of threads required?

How would you check it?

> I’ve implemented STOMP before - the protocol is easy. Should we even need this dependency?

Good! Congrats! is it in scala or java? will you open source it?
This alpakka connector basically sits on top of the particular stomp protocol. Our main objective is able to expose Stomp streams. Nevertheless, we could implement the stomp protocol in following versions of this connector to drop the depencency on vertx-stomp. The advantage of vertx-stomp is that is complete, and can handle out of the box several versions of stomp and all features.

In short: eventually we don't need this dependency, but for the moment works.

Reply:
I have no code for you unfortunately.

Profiling this would be great to get a sense of resource usage along with thread usage.

Reply:
Profiling this is not in my tool belt yet. For the sake of awareness, could you point me in the direction on where to start (blog, book, etc)? thks

Reply:
Perhaps start with [VisualVM](https://docs.oracle.com/javase/8/docs/technotes/guides/visualvm/profiler.html).